pytorch/inference/docker/2.1/py3/Dockerfile.graviton.cpu

```diff
@@ -164,7 +164,7 @@ RUN mkdir -p /etc/pki/tls/certs && cp /etc/ssl/certs/ca-certificates.crt /etc/pk
 
 # create user and folders
 RUN useradd -m model-server \
- && mkdir -p /home/model-server/tmp /opt/ml/model \
```

Comment:
I would suggest using the `${TEMP}` var.

Reply:
I thought about that but wasn't sure because the other env vars declared with TEMP at the top aren't really used in the dockerfile that way. They seem to just be for setting the environment and not intended for use in code, and that's how it was previously so I decided to just leave it